src/test/java/uk/gov/hmcts/reform/professionalapi/controller/internal/ProfessionalUserInternalControllerTest.java

```diff
@@ -116,11 +118,22 @@ public void testFindUserByEmailWithPuiUserManager() {
     }
 
     @Test
-    public void testModifyRolesForExistingUserOfOrganisation() {
+    public void testModifyRolesForExistingUserOfOrganisation() throws JsonProcessingException {
+
+        ModifyUserRolesResponse modifyUserRolesResponse = new ModifyUserRolesResponse();
+        RoleAdditionResponse roleAdditionResponse = new RoleAdditionResponse();
+        roleAdditionResponse.setIdamMessage("some nessage");
+        roleAdditionResponse.setIdamStatusCode("200");
+        modifyUserRolesResponse.setRoleAdditionResponse(roleAdditionResponse);
+        ObjectMapper mapper = new ObjectMapper();
+        String body = mapper.writeValueAsString(modifyUserRolesResponse);
+        ResponseEntity<Object> responseEntity = ResponseEntity.status(200).body(body);
+        when(professionalUserServiceMock.modifyRolesForUser(any(), any(), any())).thenReturn(responseEntity);
+
         when(userProfileUpdateRequestValidatorMock.validateRequest(userProfileUpdatedData)).thenReturn(userProfileUpdatedData);
 
         String userId = UUID.randomUUID().toString();
```

Comment:
could you please add verify for when conditions?

Reply:
Done.